### PR TITLE
Feature/dvop 2491 create new ci kafka client container

### PR DESCRIPTION
**Pull request by EloghosaU1**
-updated ci-kafka-client to have a working version 11 docker image
-built and tested locally and functionality retained
-related to Jira ticketrelated to jira ticket dvop 2491

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM alpine:latest as builder
+
+ARG KAFKA_VERSION=0.11.0.0
+ARG SCALA_VERSION=2.11
+
+RUN apk update && apk add --no-cache wget tar gzip
+
+RUN wget https://archive.apache.org/dist/kafka/0.11.0.0/kafka_${SCALA_VERSION}-0.11.0.0.tgz && \
+    tar -xzf kafka_${SCALA_VERSION}-0.11.0.0.tgz && \
+    rm kafka_${SCALA_VERSION}-0.11.0.0.tgz
```
Comment (sienkin) [inline Dockerfile]: You've defined the `KAFKA_VERSION` arg but aren't using it in the commands that follow.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+FROM amazonlinux:latest
```
Comment (sienkin) [inline Dockerfile]: We shouldn't be using the `latest` tag here. Either use `amazonlinux:2` or `amazonlinux:2023` as required.

Which one is used depends on where this image is going to be used.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+ARG KAFKA_HOME=/opt/kafka
+ARG KAFKA_VERSION=0.11.0.0
+ARG SCALA_VERSION=2.11
+ARG YUM_REPOSITORY=yum-repository.platform.aws.chdev.org
+
+ENV PATH "${PATH}:${KAFKA_HOME}/bin"
```
Comment (sienkin) [inline Dockerfile:19]: This should be `ENV PATH="${PATH}:${KAFKA_HOME}/bin"`

File: Dockerfile (hunk 2 above)
Comment (EloghosaU1) [reply Dockerfile]: okay noted

File: Dockerfile (hunk 1 above)
Comment (EloghosaU1) [reply Dockerfile]: Okay. Noted

**Review by sienkin — APPROVED** [review]

**Pull request merged**